external_loadbalancer.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+set -ex -o pipefail
+
+source common.sh
+source network.sh
+source utils.sh
+
+
+sudo firewall-cmd --zone=libvirt --add-port=6443/tcp
+sudo firewall-cmd --zone=libvirt --add-port=8080/tcp
+
+haproxy_config="${WORKING_DIR}/haproxy.cfg"
+echo $haproxy_config
+
+
+if [ "$IP_STACK" = "v6" ]
+then
+     master0=$(nth_ip $EXTERNAL_SUBNET_V6 20)
+     master1=$(nth_ip $EXTERNAL_SUBNET_V6 21)
+     master2=$(nth_ip $EXTERNAL_SUBNET_V6 22)
+     worker0=$(nth_ip $EXTERNAL_SUBNET_V6 23)
+     worker1=$(nth_ip $EXTERNAL_SUBNET_V6 24)
+else
+
+     master0=$(nth_ip $EXTERNAL_SUBNET_V4 20)
+     master1=$(nth_ip $EXTERNAL_SUBNET_V4 21)
+     master2=$(nth_ip $EXTERNAL_SUBNET_V4 22)
+     worker0=$(nth_ip $EXTERNAL_SUBNET_V4 23)
+     worker1=$(nth_ip $EXTERNAL_SUBNET_V4 24)
+fi
+
+cat << EOF > "$haproxy_config"
+defaults
+    mode                    tcp
+    log                     global
+    timeout connect         10s
```

Comment:
We should probably change this to 30. This is copy pasta from long ago that is being fixed in https://github.com/openshift/machine-config-operator/pull/1743

But that can be a followup. I'm not sure it actually makes a difference.